### ISSUE TITLE
UnusedLocalVariable false positive when variable inlined in string

### ISSUE THREAD

**Issue by avindra** [phpmd/phpmd#371]
Example:

``` php
$test = 1234;

echo "Hello ${test}";
```

This results in `$test` being reported as unused, whereas in reality it is being used.

**Comment by jdufresne** [comment]
Is this resolved by #443?

**Comment by ravage84** [comment]
Yes, it should.
